src/adapters/mc/sshpreview/refresh_for_preview.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ -n "$(command -v yum)" ]; then
+        echo "using yum"
+        yum remove openssh-server -y
+        rm /etc/ssh/sshd_config*
+        rm /etc/ssh/sshd_config.d/osconfig_*
+        yum install openssh-server -y
+
+        if [[ $(yum list installed | grep -wi osconfig) ]]; then
+                echo "OSConfig Installed, updating"
+                yum update
+                yum install osconfig
+        else
+                echo "OSConfig Not Present"
```

Comment:
nit: "installed", "not present" as all lowercase.

Also would be better to match: 

present - missing
installed - uninstalled

Reply:
updated to installed/uninstalled and to lowercase